components/Inputs/inputs.css

```diff
@@ -0,0 +1,80 @@
+@import url(/components/colors.css);
+.input-grp {
+  width: 100%;
+  margin: 1rem 0 0.5rem;
+}
+.form-label {
+  display: inline-block;
+  margin-bottom: 0.5rem;
+}
+.form-label-required::after {
+  content: "*";
+  color: var(--btn-danger-color);
+}
+.form-field {
+  display: block;
+  width: 100%;
+  padding: 0.65rem 1rem;
+  border-radius: 0.3rem;
+  border: 2px solid var(--secondary-color);
+  outline: none;
+}
+.form-field:focus {
+  box-shadow: 0 0 0 1px var(--primary-color);
```

Comment:
you can make variable for box-shadow too

Reply:
yes, I always forget that. Thanks!